src/ripple/nodestore/tests/Timing.test.cpp

```diff
@@ -360,7 +360,7 @@ class Timing_test : public beast::unit_test::suite
                     std::shared_ptr<NodeObject> result;
                     obj = seq1_.obj(dist_(gen_));
                     backend_.fetch(obj->getHash().data(), &result);
-                    suite_.expect (result && isSame(result, obj));
+                    suite_.expect(result && isSame(result, obj));
```

Comment:
member expect comes later, right?
Also, is VS removing the space or is this an intended reformat?